vehicles/src/modules/company-user-management/users/entities/user.entity.ts

```diff
@@ -49,18 +48,18 @@ export class User extends Base {
   directory: Directory;
 
   /**
-   * A property that represents the user's auth group in ORBC. It can be of types {@link UserAuthGroup},
-   * {@link ClientUserAuthGroup}, or {@link IDIRUserAuthGroup}, which are all enum types.
+   * A property that represents the user's auth group in ORBC. It can be of types {@link GenericUserAuthGroup},
+   * or {@link IDIRUserAuthGroup}, which are all enum types.
    */
   @AutoMap()
   @Column({
     type: 'simple-enum',
-    enum: [UserAuthGroup, ClientUserAuthGroup, IDIRUserAuthGroup],
+    enum: [GenericUserAuthGroup, IDIRUserAuthGroup],
     length: 10,
     name: 'USER_AUTH_GROUP_TYPE',
     nullable: true,
   })
-  userAuthGroup: UserAuthGroup | ClientUserAuthGroup | IDIRUserAuthGroup;
+  userAuthGroup: GenericUserAuthGroup | IDIRUserAuthGroup;
```

Comment:
I'm confused here. As I understand, PUBLIC_VERIFIED is a classification of userAuthGroup. So, how can userAuthGroup be ANONYMOUS or PUBLIC? Shouldn't it be PAPPLICANT or ORGADMIN, currently?

Reply:
Need to insert PUBLIC in ORBC_USER for non staff user for time being due to design. Might be revisited when discovery starts on basic BCeID etc according to the architect